src/charClassIcons.js

```diff
@@ -0,0 +1,31 @@
+import squizard from './assets/squizard.jpeg';
+import barbarian from './assets/barbarian.svg';
+import bard from './assets/bard.svg';
+import cleric from './assets/cleric.svg';
+import druid from './assets/druid.svg';
+import fighter from './assets/fighter.svg';
+import monk from './assets/monk.svg';
+import paladin from './assets/paladin.svg';
+import ranger from './assets/ranger.svg';
+import rogue from './assets/rogue.svg';
+import sorcerer from './assets/sorcerer.svg';
+import warlock from './assets/warlock.svg';
+import wizard from './assets/wizard.svg';
+
+const charClassIcons = {
+  squizard: squizard,
+  barbarian: barbarian,
+  bard: bard,
+  cleric: cleric,
+  druid: druid,
+  fighter: fighter,
+  monk: monk,
+  paladin: paladin,
+  ranger: ranger,
+  rogue: rogue,
+  sorcerer: sorcerer,
+  warlock: warlock,
+  wizard: wizard
+}
+
+export default charClassIcons;
```

Comment:
This file exists only as a way to keep the icon imports out of another file.  Probably should move this to `/assets/` though?

Reply:
I'm wondering if we should remove / not render the five non-magical classes, or add a message when you click on one them: "Squizzard says sorry, you're not allowed to do magic."